sdk/include/opentelemetry/sdk/trace/simple_processor.h

```diff
@@ -57,6 +62,7 @@ class SimpleSpanProcessor : public SpanProcessor
          * logged in this case. */
       }
     }
+#ifdef ENABLE_ASYNC_EXPORT
```

Comment:
Should `if (is_export_async_ == false)` only be checked when `ENABLE_ASYNC_EXPORT` is defined?

Reply:
Done